### PR TITLE
Remove EngineManagerStandalone

### PR DESCRIPTION
**Pull request by kitchoi**
Originally `EngineManagerStandalone` was meant to replicate the functions of the `EngineManagerStandaloneGUI` or `EngineManagerMayavi2` without the use of a GUI.
However, since changes in the `__init__` of CUDSSource, EngineSource made creating a Source more user-friendly plus the fact that the `EngineManager*` has not grown as big as was anticipated, there is little gain now to keep `EngineManagerStandalone` around.  Therefore proposing getting rid of it.

Also, "fix" 1/3 of the issue in #172 (by removing the problem....hmm)

@itziakos, @stefanoborini

### CODE REVIEW

**Comment by kitchoi** [comment]
Not yet done with cleaning the doc for this yet.  Want to know what you guys think.

**Comment by stefanoborini** [comment]
:+1: for me.

**Comment by kitchoi** [comment]
@itziakos, what do you think?  if this is good I will go on to clean the related documentation before removing the WIP tag.

**Comment by itziakos** [comment]
Yes, please move forward with the changes

**Comment by stefanoborini** [comment]
What's the status of this now?

**Comment by kitchoi** [comment]
@stefanoborini I believe this only needs updating the documentation.

**Comment by stefanoborini** [comment]
Should I get rid of the examples, or convert them to a different strategy? (it's unclear to me how to do non-gui, considering that now you seem to need a panel to achieve animations, as far as I understand from the tests)

**Comment by kitchoi** [comment]
@stefanoborini The batch scripting approach (already in the doc) with some modification using [`mlab.animate`](http://docs.enthought.com/mayavi/mayavi/auto/mlab_other_functions.html?highlight=animate#animate) should be enough to achieve animation.  In my opinion, it is fine to just get rid of the current example and rely on mayavi's doc.  Providing an example using `mlab.animate` would be being-kind-to-new-users :)

**Comment by stefanoborini** [comment]
@kitchoi please review. There is apparently a change in behavior with most up-to-date packages, so I had to fix the versions.

File: doc/source/conf.py [hunk 1]
```diff
@@ -3,6 +3,10 @@
 # SimPhoNy-Mayavi documentation build configuration file
 #
 import sys
+import os
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__name__), '..', '..')))
```
Comment (kitchoi) [inline doc/source/conf.py:9]: This is for importing `simphony_mayavi` without having installed it (or setup.py develop) beforehand?

**Comment by kitchoi** [comment]
Thanks @stefanoborini 
Is it possible to skip a particular version of a particular package (e.g. `flake8 != 3.0.0`) instead of fixing the versions for all packages?

File: doc/source/conf.py (hunk 1 above)
Comment (stefanoborini) [inline doc/source/conf.py:9]: Even when setup develop, it still can't find it. Do you have a PYTHONPATH defined by any chance?

Comment (kitchoi) [inline doc/source/conf.py:9]: No, never.

**Comment by stefanoborini** [comment]
Question is: is it good practice to leave the version unspecified? I understood it wasn't from previous experiences.

**Comment by kitchoi** [comment]
There has been a similar discussion on https://github.com/simphony/simphony-framework/issues/15
My own preference is to skip known-problematic versions, this makes dependency resolution a lot easier.

File: doc/source/conf.py (hunk 1 above)
Comment (stefanoborini) [inline doc/source/conf.py:9]: I don't know what to say. Without that it doesn't work.

**Comment by stefanoborini** [comment]
The test failure is due to this commit in mayavi

```
f3fd83b57137b3ac09309ffbb4d3c529352265b2 is the first bad commit
commit f3fd83b57137b3ac09309ffbb4d3c529352265b2
Author: Prabhu Ramachandran <prabhu@aero.iitb.ac.in>
Date:   Sun Jul 3 00:38:06 2016 -0400

    Support recording movie with animate decorator.

    If one has a scene with the movie_maker.record set to True, a movie will
    be recorded when the animation is run.

:040000 040000 1755a2f37a626855627403309e850f13527ecb2d ca0ac36cda6e985e61f581e4fce01c6add49197b M  mayavi
:040000 040000 7a9b3c7c6367ddce6dd3bc01b8bd7e8dac88d655 92b0c79f5cac9591530d59f0f6838b7a7addbfcd M  tvtk
```

**Comment by stefanoborini** [comment]
@kitchoi any comment on this?

File: simphony_mayavi/plugins/tests/test_run_and_animate_panel.py [hunk 2]
```diff
@@ -42,6 +40,7 @@ def test_animate_fired(self):
         ui = self.panel.show_config()
 
         def engine_ran_twice(source):
+            print("XXXXXXXXXXXXXXXXXXXXXXXXX", source.engine.time)
```
Comment (stefanoborini) [inline simphony_mayavi/plugins/tests/test_run_and_animate_panel.py]: remove

**Comment by stefanoborini** [comment]
After discussion with @itziakos we agreed to do the following for the current red:
- Pin to mayavi 4.4.4
- Open a mayavi issue relative to 4.5.0 behavior.

**Pull request merged**